controllers/articleRatingController.js

```diff
@@ -0,0 +1,63 @@
+import models from '../models';
+import Response from '../helpers/responseHelper';
+import { STATUS } from '../helpers/constants';
+import getAverageRatings from '../helpers/articleRatingHelper';
+
+const { ratings } = models;
+/**
+ * Class representing article rating controller.
+ *
+ * @export
+ * @class ArticleRatingController
+*/
+export default class ArticleRatingController {
+  /**
+   * Sends the request payload to the database and returns the article object
+   * @static
+   * @param {function} req the request object
+   * @param {function} res the resposne object
+   * @returns {function} the Article object
+   */
+  static async create(req, res) {
+    const userId = req.user.id;
+    const { articleId } = req.params;
+    const { body: { stars } } = req;
+    try {
+      const articleRating = await ratings.create({
+        userId,
+        articleId,
+        stars
+      });
+      return Response.send(res, STATUS.CREATED, articleRating, 'success');
+    } catch (err) {
+      return Response.send(res, STATUS.SERVER_ERROR, err, 'failed');
+    }
+  }
+
+  /**
+   * Sends the request payload to the database and returns the article object
```

Comment:
#### Nitpick
- Use much more descriptive JS-DOC.

Reply:
@aanchirinah Okay I will do that.